RecoTracker/SiTrackerMRHTools/interface/SimpleDAFHitCollector.h

```diff
@@ -16,11 +16,12 @@ class StripRecHit1D;
 
 class SimpleDAFHitCollector :public MultiRecHitCollector {
 	public:
-	explicit SimpleDAFHitCollector(const MeasurementTracker* measurementTracker,
+	explicit SimpleDAFHitCollector(const TrackerTopology* trackerTopology,
+                                 const MeasurementTracker* measurementTracker,
 				 const SiTrackerMultiRecHitUpdator* updator,
 			         const MeasurementEstimator* est,
 				 const Propagator* propagator, bool debug
-				 ):MultiRecHitCollector(measurementTracker), theUpdator(updator), theEstimator(est), thePropagator(propagator), debug_(debug){
+				 ):MultiRecHitCollector(measurementTracker), theTopology(trackerTopology), theUpdator(updator), theEstimator(est), thePropagator(propagator), debug_(debug){
```

Comment:
I think it's better style to first initialize the base classes (even if technically it shouldn't matter). I made `TrackerTopology` the first argument because it is a more 'basic' thing than the other arguments.